llm_bench/python/requirements.txt

```
@@ -2,6 +2,7 @@
 numpy
 openvino>=2024.1.0
 openvino_tokenizers>=2024.1.0
+openvino_genai>=2024.2.0
```

for alignment versions:

```suggestion
openvino>=2024.2.0
openvino_tokenizers>=2024.2.0
openvino_genai>=2024.2.0
```